### DISCUSSION TITLE
[extra740] Resources more than max items

### DISCUSSION THREAD

**Discussion by MxxxxxF** [prowler-cloud/prowler#1399] [Q&A] [answered]
Hi Prowler, 

Regarding to this check rule [extra740] Check if EBS snapshots are encrypted, I have about 500+ EBS snapshots. 
However, based on my output, the scan only report my first 100 snapshots. 
Is this a limitation?
How can I have prowler to scan all of my snapshots? 

Current the output display as below
PASS - 99. snapshot_99 is encrypted 
PASS - 100.snapshot_100 is encrypted
PASS - 101. 500 encrypted snapshots out of 500 snapshots found. Only the first 100 encrypted snapshots are reported.
INFO  - 102. 0 unencrypted EBS volume snapshots out of 500 total snapshots

Thank you

**Comment by jfagoagas** [comment, marked as answer]
Hi @MxxxxxF, thank you for using Prowler.

I've created an issue because there is some limitation within the check's logic. We're going to check it and we'll ket you know.

Thanks!

**Reply by jfagoagas** [reply]
Hi @MxxxxxF we fix it refactoring the logic to show one line per resource and getting rid of the % value because it doesn't follow Prowler's checks pattern.

You can get more information here:
- Issue: https://github.com/prowler-cloud/prowler/issues/1400
- PR: https://github.com/prowler-cloud/prowler/pull/1405

Thank you for you contribution.
